src/decrypter.js

```diff
@@ -290,75 +341,77 @@ decrypt = function(encrypted, key, initVector) {
   return decrypted;
 };
 
-AsyncStream = function() {
-  this.jobs = [];
-  this.delay = 1;
-  this.timeout_ = null;
-};
-AsyncStream.prototype = new videojs.Hls.Stream();
-AsyncStream.prototype.processJob_ = function() {
-  this.jobs.shift()();
-  if (this.jobs.length) {
-    this.timeout_ = setTimeout(this.processJob_.bind(this),
-                               this.delay);
-  } else {
+export class AsyncStream extends Stream {
+  constructor() {
+    super(Stream);
+    this.jobs = [];
+    this.delay = 1;
     this.timeout_ = null;
   }
-};
-AsyncStream.prototype.push = function(job) {
-  this.jobs.push(job);
-  if (!this.timeout_) {
-    this.timeout_ = setTimeout(this.processJob_.bind(this),
-                               this.delay);
+  processJob_() {
+    this.jobs.shift()();
+    if (this.jobs.length) {
+      this.timeout_ = setTimeout(this.processJob_.bind(this),
+                                 this.delay);
+    } else {
+      this.timeout_ = null;
+    }
   }
-};
+  push(job) {
+    this.jobs.push(job);
+    if (!this.timeout_) {
+      this.timeout_ = setTimeout(this.processJob_.bind(this),
+                                 this.delay);
+    }
+  }
+}
+
+// the maximum number of bytes to process at one time
+const DecrypterStep = 4 * 8000;
```

Comment:
`DECRYPTER_STEP`. Also, can this just be declared and defined as `Decrypter.STEP`?